rules/renaming/tests/Rector/FuncCall/RenameFunctionAutoimportRector/RenameFunctionAutoimportRectorTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Renaming\Tests\Rector\FuncCall\RenameFunctionAutoimportRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class RenameFunctionAutoimportRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        $this->doTestFileInfo($fileInfo);
```

Comment:
Could you verify the 'Symfony\Component\DependencyInjection\Loader\Configurator\service' is loaded?



```php
$this->assertTrue(function_exists('...'));
```

Reply:
assert added